fuel-merkle/src/common/position.rs

```diff
@@ -55,7 +62,7 @@ use core::convert::Infallible;
 /// This allows us to construct a `Position` (and its in-order index) by
 /// providing either an in-order index directly or, in the case of a leaf, a
 /// leaf index. This functionality is captured by `from_in_order_index()` and
-/// `from_leaf_index()` respectively.
+/// `from_leaf_index_unwrap()` respectively.
```

Comment:
I modified comment, sorry=D

```suggestion
/// `from_leaf_index()` respectively.
```